Cargo.toml

```diff
@@ -1,12 +1,17 @@
 [package]
 name = "qdrant"
 version = "0.11.1"
-authors = ["Andrey Vasnetsov <andrey@vasnetsov.com>"]
+authors = [
+    "Andrey Vasnetsov <vasnetsov93@gmail.com>",
```

Comment:
why you changed my email?)

Reply:
Whoops. This one was pulled from git commits. Reverting.